### PR TITLE
test(fsObs): Test different fs observable executions.

### PR DESCRIPTION
**Pull request by jdetle**
Need to add more mocks

### CODE REVIEW

**Comment by jdetle** [comment]
@rgbkrk @lgeiger @captainsafia, ready for a revue

File: test/renderer/api/kernel-spec.js [hunk 1]
```diff
@@ -31,26 +31,32 @@ const emptyKernel = Object.freeze({
...
 }
 
 describe('forceShutdownKernel', () => {
+  var sandbox;
+  beforeEach(function() {
+    sandbox = sinon.sandbox.create();
+  });
+  afterEach(function() {
+    sandbox.restore();
+  });
```
Comment (lgeiger) [inline test/renderer/api/kernel-spec.js:56]: Would it work if we move `beforeEach` and `afterEach` to the outside of the `describe` function? This way we don't have to repeat them.

**Review by lgeiger — APPROVED** [review]
Nice 🎉

File: test/renderer/api/kernel-spec.js (hunk 1 above)
Comment (rgbkrk) [reply test/renderer/api/kernel-spec.js:56]: In this case, in tests, I think it's fine to repeat the setup code for the `beforeEach` and `afterEach`.

**Review by rgbkrk — APPROVED** [review]

**Pull request merged**